TaskManager/Modules/Home/View/TaskTableViewCell.swift

```diff
@@ -35,27 +35,16 @@ class TaskTableViewCell: UITableViewCell {
         descriptionLabel.numberOfLines = 0
     }
     
-    private func changeCompletedTask() {
+    func changeCompletedTask() {
         let attributeString: NSMutableAttributedString = NSMutableAttributedString(string: titleLabel.text ?? "")
         attributeString.addAttribute(NSAttributedString.Key.strikethroughStyle, value: 2, range: NSRange(location: 0, length: attributeString.length))
         titleLabel.attributedText = attributeString
         titleLabel.textColor = .completedColor
     }
     
-    func configureCell(viewModel: ModelTask) {
+    func configureCell(viewModel: ModelTask, backgroundColor: UIColor) {
         titleLabel.text = viewModel.title
-        descriptionLabel.text = viewModel.description
-        
-        let section = viewModel.type
-        
-        switch section {
-        case .Active:
-            backgroundColor = .systemBackground
-            circleLabel.backgroundColor = backgroundColor
-        case .Completed:
-            backgroundColor = .systemBackground
-            circleLabel.backgroundColor = .taskManagerColor
-            changeCompletedTask()
-        }
+        descriptionLabel.text = viewModel.subtitle
```

Comment:
you can still keep the logic here with viewModel.type

if viewModel.type {
...
} else {
...
}